app/views/reports/index.html.erb

```diff
@@ -18,6 +18,15 @@
           - for census sign off
         </p>
       </li>
+    
+      <% if !current_user.system_admin? && !current_user.lead_school? %>
```

Comment:
You can use the pundit policy scope here. Something like:
```suggestion
      <% if policy(:user).bulk_qts? %>
```
https://github.com/varvet/pundit#headless-policies